src/style.css

```diff
@@ -1,3 +1,83 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  width: 100%;
+  background-color: white;
+  position: relative;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: right;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+  margin-right: 50px;
+}
+
+.moyo-logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  padding: 0;
+  margin: 0;
+
+}
+
+.nav-link {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  line-height: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  height: 15px;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+}
+
+a:hover {
+  color: #00acdc;
+}
+
+.item-1 {
```

Comment:
you may use [:first-child ](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child) structural pseudo-class to do it and remove `item-1` class

Reply:
It doesnt work, if I do with :first-child, than all links become blue